### PR TITLE
pypy 2.6.1

### PR DESCRIPTION
**Pull request by alex**

### CODE REVIEW

File: Library/Formula/pypy.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Pypy < Formula
   desc "Implementation of Python 2 in Python"
   homepage "http://pypy.org/"
-  url "https://bitbucket.org/pypy/pypy/downloads/pypy-2.6.0-src.tar.bz2"
-  sha256 "9bf353f22d25e97a85a6d3766619966055edea1ea1b2218445d683a8ad0399d9"
+  url "https://bitbucket.org/pypy/pypy/downloads/pypy-2.6.1-src.tar.bz2"
+  sha256 "7fddd414c9348c2f899f79ad86adc3fc2b19443855b5243f58487e1f0ac46560"
   revision 1
```
Comment (DomT4) [inline Library/Formula/pypy.rb]: Please cut this line on version bumps, Thanks!

**Comment by alex** [comment]
@DomT4 doh, sorry I missed that!

**Comment by tdsmith** [comment]
Thanks Alex! Pip and setuptools didn't get bumped but I'd rather not rebuild. Maybe I should split those out of the bottle. :/

**Comment by apjanke** [comment]
This is still [using the 2.5.0 osx bootstrap](https://github.com/Homebrew/homebrew/blob/02270bfe85f159af72e92013b1caced1475ee259/Library/Formula/pypy.rb#L25-L28) even though a 2.6.1 on is out on the [`pypy` download page](https://bitbucket.org/pypy/pypy/downloads/). Should that be bumped too?

```
  resource "bootstrap" do
    url "https://bitbucket.org/pypy/pypy/downloads/pypy-2.5.0-osx64.tar.bz2"
    sha256 "30b392b969b54cde281b07f5c10865a7f2e11a229c46b8af384ca1d3fe8d4e6e"
  end
```

**Comment by DomT4** [comment]
I don't think the bootstrap version matters *too much*, but Tim's the boss.

**Comment by tdsmith** [comment]
I wouldn't change it unless it's causing problems.

**Comment by apjanke** [comment]
No problems observed for me; was just wondering.

**Pull request closed without merging**